config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   devise_for :admins
   resources :organizations
-
+  get "/pending", to: "organizations#pending"
```

Comment:
if you add `, as: :pending_organizations` to the end of this line, we'll be able to use `pending_application_path` as a URL helper in the app.